src/codegen/fast_math.hpp

```diff
@@ -113,7 +103,7 @@ static inline double egm1(double x, double px) {
 /// double precision exp function
 static inline double vexp(double initial_x) {
     double x = initial_x;
-    double px = dpfloor(LOG2E * x + 0.5);
+    double px = std::floor(LOG2E * x + 0.5);
```

Comment:
so `dpfloor` and `spfloor` can be simply replaced by `std::floor` ?

Reply:
as far as I can tell, they do exactly the same. The only concern I have is the ability of the compiler to vectorize either option.

Reply:
looks same to me.